assets/huggingface.yaml

```diff
@@ -141,3 +141,25 @@
   prohibited_uses: ''
   monitoring: unknown
   feedback: https://huggingface.co/TurkuNLP/bloom-finnish-176b/discussions
+- type: dataset
+  name: Cosmopedia v0.1
+  organization: Hugging Face
+  description: Cosmopedia is a dataset of synthetic textbooks, blogposts, stories, posts, and WikiHow articles generated by Mixtral-8x7B-Instruct-v0.1. The dataset contains over 30 million files and 25 billion tokens, making it the largest open synthetic dataset to date. It covers a variety of topics, mapping worldwide knowledge from Web datasets like RefinedWeb and RedPajama, to generate synthetic content.
+  created_date: 2024-02-22
+  url: none
+  datasheet: https://huggingface.co/datasets/HuggingFaceTB/cosmopedia
+  modality: text
+  size: 30 million files and 25 billion tokens
+  sample: []
+  analysis: Some seed samples were used in different prompt styles and audiences. Less than 1% of files are duplicates after running MinHash deduplication. Contaminated samples were removed from each dataset split.
+  dependencies: [RefinedWeb, RedPajama-Data, Mixtral, Phi-1.5]
```

Comment:
is this right?

Reply:
should just be Mixtral, GPT included the others since they were mentioned (but not as dependencies)